python/test/scripts/example.py

```diff
@@ -45,7 +47,36 @@ def main(args):
     # - 2017-03-03T06:00:00.000-05:00  (timezone specified)
     # - 2017-03-03T06:00:00-05:00      (timezone specified)
     ##
-    event_time = datetime(year = 2017, month = 3, day = 3, hour = 6, minute = 0, tzinfo = timezone('US/Eastern'))
+    event_time = datetime(
+        year = 2017, month = 3, day = 3, hour = 6, minute = 0, tzinfo = timezone('US/Eastern')
+    )
+
+    vms = [
+        CloudVM(uuid = "421bac04-f3d7-a600-ef5c-b3219e90d1eb"),
+        CloudVM(
+            network_interfaces = [
+                NetworkInterface(mac_address = "00:50:56:9b:7c:31"),
+                NetworkInterface(mac_address = "00:50:56:9b:06:1c"),
+            ],
+        ),
+        CloudVM(
+            network_interfaces = [
+                NetworkInterface(ip_addresses = [
+                    "fe80::250:56ff:fe9b:5945", "10.17.109.123",
+                ]),
+            ],
+        ),
+        CloudVM(
+            network_interfaces = [
```

Comment:
please maintain the property (while in python standard way) closer to the our API model to avoid the disconnect.